runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -532,30 +528,25 @@ ifneq ($(JITSERVER_SUPPORT),)
     SOLINK_SLINK_STATIC=-l:libprotobuf.a
     CXX_DEFINES+=GOOGLE_PROTOBUF_NO_RTTI
 
-    ifneq ($(JITSERVER_ENABLE_SSL),)
-        SOLINK_SLINK+=ssl
-
-        ifneq ($(OPENSSL_CFLAGS),)
-            ADD_OPENSSL_CFLAGS=false
-            # --with-openssl=system, OPENSSL_LIBS looks like "-L/path/to/system/openssllib -lssl -lcrypto"
-            ifneq ($(OPENSSL_LIBS),)
-                SOLINK_LIBPATH+=$(subst -L,,$(firstword $(OPENSSL_LIBS)))
-                ADD_OPENSSL_CFLAGS=true
-            # --with-openssl=fetched --enable-openssl-bundling
-            else ifneq ($(OPENSSL_BUNDLE_LIB_PATH),)
-                SOLINK_LIBPATH+=$(OPENSSL_BUNDLE_LIB_PATH)
-                SOLINK_FLAGS+=-Wl,-rpath,\$$ORIGIN/..
-                ADD_OPENSSL_CFLAGS=true
-            # --with-openssl=fetched only
-            else ifneq ($(OPENSSL_DIR),)
-                SOLINK_LIBPATH+=$(OPENSSL_DIR)
-                ADD_OPENSSL_CFLAGS=true
-            endif
-
-            ifeq (true, $(ADD_OPENSSL_CFLAGS))
-                C_INCLUDES+=$(subst -I,,$(OPENSSL_CFLAGS))
-                CXX_INCLUDES+=$(subst -I,,$(OPENSSL_CFLAGS))
-            endif
-        endif # OPENSSL_CFLAGS
-    endif # JITSERVER_ENABLE_SSL
+    SOLINK_SLINK+=ssl
+
+    ifneq ($(OPENSSL_CFLAGS),)
+        ADD_OPENSSL_CFLAGS=false
+        # --with-openssl=fetched --enable-openssl-bundling
+        ifneq ($(OPENSSL_BUNDLE_LIB_PATH),)
+            SOLINK_SLINK+=crypto
+            SOLINK_LIBPATH+=$(OPENSSL_BUNDLE_LIB_PATH)
+            SOLINK_FLAGS+=-Wl,-rpath,\$$ORIGIN/..
+            ADD_OPENSSL_CFLAGS=true
+        # --with-openssl=fetched only
+        else ifneq ($(OPENSSL_DIR),)
+            SOLINK_LIBPATH+=$(OPENSSL_DIR)
+            ADD_OPENSSL_CFLAGS=true
+        endif
+
+        ifeq (true, $(ADD_OPENSSL_CFLAGS))
+            C_FLAGS+=$(OPENSSL_CFLAGS)
+            CXX_FLAGS+=$(OPENSSL_CFLAGS)
+        endif
+    endif # OPENSSL_CFLAGS
```

Comment:
I suggest that `OPENSSL_CFLAGS` should be added to `C_FLAGS` and `CXX_FLAGS` if `OPENSSL_CFLAGS` is not empty, independent of handling of `OPENSSL_BUNDLE_LIB_PATH` and `OPENSSL_DIR`.

Reply:
Good point, `OPENSSL_CFLAGS` could be set up by `pkg-confg` even when `OPENSSL_DIR` and `OPENSSL_BUNDLE_LIB_PATH` are not set up. Will make the change